packages/jellyfish-api-core/__tests__/category/blockchain/getChainTips.test.ts

```diff
@@ -9,6 +9,7 @@ describe('ChainTips', () => {
   beforeAll(async () => {
     await container.start()
     await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
```

Comment:
```suggestion
    await container.waitForBlock(1)
```

This will make the test run faster.